triedb/hashdb/database.go

```diff
@@ -449,12 +454,15 @@ func (db *Database) Commit(node common.Hash, report bool) error {
 	memcacheCommitBytesMeter.Mark(int64(storage - db.dirtiesSize))
 	memcacheCommitNodesMeter.Mark(int64(nodes - len(db.dirties)))
 
+	memcacheCommitStandardTimer.Update(time.Since(start))
+
 	logger := log.Info
 	if !report {
 		logger = log.Debug
 	}
 	logger("Persisted trie from memory database", "nodes", nodes-len(db.dirties)+int(db.flushnodes), "size", storage-db.dirtiesSize+db.flushsize, "time", time.Since(start)+db.flushtime,
 		"gcnodes", db.gcnodes, "gcsize", db.gcsize, "gctime", db.gctime, "livenodes", len(db.dirties), "livesize", db.dirtiesSize)
+	logger("Persisted trie from memory database", "time_since_start", time.Since(start))
```

Comment:
can we append the `time_since_start` to the previous logger call?
Or even better, could we split `"time", time.Since(start)+db.flushtime` into `"time", time.Since(start)` and `"flushtime", db.flushtime`? And then we could split in the same way "nodes" and "size".

Reply:
Done.
I think it is easier to maintain a fork repo if we are more inclined to not modify lines of code, compared to only adding new lines of codes.
That is why I initially added an extra logger.
But since since this is a quite simple case I think it is OK to modify the existing line of code :)